frontend/src/app/(editor)/@appBar/login/page.tsx

```diff
@@ -1,11 +1,29 @@
+'use client'
 /*
  * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
+
 import React from 'react'
-import { BaseAppBar } from '../../../../components/layout/app-bar/base-app-bar'
+import { Nav, Navbar } from 'react-bootstrap'
+import styles from '../../../../components/layout/app-bar/navbar.module.scss'
+import { HelpDropdown } from '../../../../components/layout/app-bar/app-bar-elements/help-dropdown/help-dropdown'
+import { SettingsButton } from '../../../../components/global-dialogs/settings-dialog/settings-button'
+import { NewNoteButton } from '../../../../components/common/new-note-button/new-note-button'
+import { HistoryButton } from '../../../../components/layout/app-bar/app-bar-elements/help-dropdown/history-button'
 
 export default function AppBar() {
-  return <BaseAppBar>Login</BaseAppBar>
+  return (
+    <Navbar expand={true} className={`px-2 py-1 align-items-center ${styles.navbar}`}>
+      <Nav className={`align-items-stretch justify-content-end flex-grow-1 ${styles.side} h-100 py-1`}>
+        <div className={'d-flex gap-2'}>
+          <HistoryButton />
+          <HelpDropdown />
+          <SettingsButton />
+          <NewNoteButton />
+        </div>
+      </Nav>
+    </Navbar>
+  )
```

Comment:
This is a duplicate of the code in `base-app-bar.tsx`, isn't it? Is this intended? If yes why is this necessary?

Reply:
No. This one has no branding, no hedgedoc icon and no user menu.